src/app/beer_garden/events/processors.py

```diff
@@ -13,13 +14,15 @@
 class BaseProcessor(StoppableThread):
     """Base Processor"""
 
-    def __init__(self, action=None, **kwargs):
+    def __init__(self, action=None, black_list=list(), **kwargs):
         super().__init__(**kwargs)
 
         self._action = action
+        self._black_list = black_list
```

Comment:
Two minor things:
1. It's standard to use the python syntax for empty arrays (and dicts) as opposed to the constructor-style.
2. It's not a good idea to have a mutable type be the default value for a parameter.

So this would be better as:
```python
def __init__(self, ..., black_list=None, ...):
   ...
   self._black_list = black_list or []
```